tests/installation/grub_test.pm

```diff
@@ -100,7 +102,7 @@ sub run {
     }
     else {
         # avoid timeout for booting to HDD
-        send_key 'ret';
+        send_key 'ret' for (1 .. $retry);
```

Comment:
I wouldn't call this $retry - it's just a loop without any expectations in between. And if the key is not lost, you just added 4 additional enter presses into the keyboard buffer. You will have fun finding the fallout of those :)

Reply:
damn, i thought no one will notice ;)

Reply:
As okurz keeps a record of my quotes - ask him for the Pappenheimer :)